### PR TITLE
Farewell, Ethereum 1

### PR DESCRIPTION
**Pull request by AurevoirXavier**
Remove Ethereum 1 related code.

Also, a part of #1157.

### CODE REVIEW

File: primitives/ethereum/Cargo.toml [hunk 1]
```diff
@@ -35,6 +35,7 @@ ethereum-types = { version = "0.12", features = ["codec"] }
 keccak-hasher  = { version = "0.15" }
```
Comment (AsceticBear) [inline primitives/ethereum/Cargo.toml:35]: After this pr,  we can delete this crate IMO. Only used  as `use ethereum_primitives::{H160, H256, U256};`. There are many replacement solutions.

Comment (AurevoirXavier) [reply primitives/ethereum/Cargo.toml:35]: #1456

Comment (AsceticBear) [reply primitives/ethereum/Cargo.toml:35]: Why not do it in this PR together?

Comment (AurevoirXavier) [reply primitives/ethereum/Cargo.toml:35]: I prefer to open another PR to do the cleaning job.

**Review by AsceticBear — APPROVED** [review]

**Review by HackFisher — APPROVED** [review]

**Pull request merged**